acceptancetests/utility.py

```diff
@@ -297,6 +299,9 @@ def add_basic_testing_arguments(
     parser.add_argument('--keep-env', action='store_true',
                         help='Keep the Juju environment after the test'
                         ' completes.')
+    parser.add_argument('--logging-config',
+                        help="Override logging configuration for a deployment.",
+                        default="<root>=INFO;unit=INFO")
```

Comment:
Didn't I add this originally, or am I going mental?

Reply:
Yes, but that was in develop. In my #9841 changes I forgot to include the file so all other tests started failing because of the missing arg.